src/sentry/integrations/repositories.py

```diff
@@ -49,12 +50,16 @@ def get_stacktrace_link(self, repo, filepath, default, version):
         If no file was found return `None`, and re-raise for non "Not Found" errors
 
         """
-        if version:
-            source_url = self.check_file(repo, filepath, version)
-            if source_url:
-                return source_url
-
-        source_url = self.check_file(repo, filepath, default)
+        with configure_scope() as scope:
+            scope.set_tag("stacktrace_link.tried_version", False)
+            if version:
+                scope.set_tag("stacktrace_link.tried_version", True)
```

Comment:
@MeredithAnya are you sure this will wipe out the value set two lines above?

Reply:
@scefali testing this out locally it did replace the previously set tag value